backend/data_access/S3.js

```diff
@@ -0,0 +1,131 @@
+import AWS from "aws-sdk";
```

Comment:
This is okay for now. `aws-sdk` is the version 2 of the SDK. It is highly encouraged that we use the V3 of the SDK, which can be found here: https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-s3/index.html